examples/supermassive-todomvc/package.json

```diff
@@ -31,7 +31,7 @@
     "uuid": "^8.3.2",
     "webpack": "^5.76.0",
     "webpack-cli": "^4.6.0",
-    "webpack-dev-server": "^3.1.14",
+    "webpack-dev-server": "^4.13.3",
```

Comment:
nitpick: I guess we can move it do devDependencies.

Reply:
Thanks! I've moved it :)